modules/virt-importing-vm-prerequisites.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
+// * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
+[id='virt-importing-vm-prerequisites_{context}']
+= Prerequisites for importing a virtual machine into {VirtProductName}
+
+ifdef::virt-importing-rhv-vm[]
+Importing a Red Hat Virtualization virtual machine into {VirtProductName} has the following prerequisites:
+
+* If the operating system of the virtual machine does not map to a common {VirtProductName}, you must create a custom ConfigMap to map the operating system to a common {VirtProductName} template.
```

Comment:
The first instance of `common {VirtProductName}` -> `common {VirtProductName} template`, right?

Reply:
Right! Will fix.